release.py

```diff
@@ -313,6 +316,9 @@ def commit_type_to_category(commit_type: str) -> str:
         'docs': 'Documentation',
         'test': 'Tests',
         'ci': 'CI',
+        'perf': 'Performance',
+        'refactor': 'Refactor',
+        'revert': 'Revert',
```

Comment:
Perhaps this, as per the (now hidden) thread:

```suggestion
        'refactor': 'Refactoring',
        'revert': 'Reverted',
```

Reply:
I quite like that (Refactoring and Reverted)